DemoApp/DemoApp/TMButtonPage.xaml.cs

```diff
@@ -1,16 +1,104 @@
+using Trimble.Modus.Components;
+using Trimble.Modus.Components.Enums;
+using Size = Trimble.Modus.Components.Enums.Size;
+
 namespace DemoApp;
 
 public partial class TMButtonPage : ContentPage
 {
-	public TMButtonPage()
+    private string _selectedStyle ="Fill";
+    private string _selectedSize ="Default";
+    private bool isDiabled;
```

Comment:
Please rename this property with _isDisabled. Since other private properties follow that pattern

Reply:
Renamed